### ISSUE TITLE
No "clj-suitable" autocompletion candidates for "js/" with cider-jack-in-cljs

### ISSUE THREAD

**Issue by haoyuan80s** [clojure-emacs/cider#2871]
I am trying to get autocompletion candidates after for` js/ `. Following the information in https://github.com/rksm/clj-suitable, I am able to get the candidates with the with `clj -A:suitable` (results shown in "expected behavior" section). But, I cannot make it work with cider. 

## Expected behavior
![image](https://user-images.githubusercontent.com/9064146/86185086-605c5980-baea-11ea-9c9b-3ad5eb99494f.png)

## Actual behavior
![image](https://user-images.githubusercontent.com/9064146/86185584-86362e00-baeb-11ea-8d79-5abb1398df9f.png)

## Steps to reproduce the problem
```
lein new figwheel-main example -- --reagent
cider-jack-in-cljs --> figwheel-main --> dev
```

### CIDER version information
```
CIDER 0.26.0snapshot (package: 20200629.944)
java version "12.0.2" 2019-07-16
```

### Lein/Boot version
`Leiningen 2.9.3 on Java 12.0.2 Java HotSpot(TM) 64-Bit Server VM`
### Emacs version
`GNU Emacs 28.0.50 (build 1, x86_64-apple-darwin18.7.0, NS appkit-1671.60 Version 10.14.6 (Build 18G2022)) of 2020-02-14
`
### Operating system
`MacOS Mojave version 10.14.6
`

**Comment by bbatsov** [comment]
Dynamic completion with suitable is currently disabled due to issues with `shadow-cljs`. See https://github.com/rksm/clj-suitable/issues/15

**Comment by haoyuan80s** [comment]
Thanks. I downgrade cider to the stable version 0.25.0. It works well.

**Comment by prantor19** [comment]
@haoyuan80s How can I downgrade cider?

**Comment by prantor19** [comment]
@haoyuan80s How can I downgrade cider? Installed version is cider-20200916.1152

**Comment by bbatsov** [comment]
See https://metaredux.com/posts/2019/07/10/hard-cider-easy-downgrading.html

Generally, I wouldn't recommend this, though. As noted the problem is an external dependency, so I was left in a tricky situation as it works for figwheel, but breaks completely for shadow-cljs.

**Comment by bbatsov** [comment]
That's fixed in suitable 0.4 and cider-nrepl 0.26.
